text/103-leaving-x-twitter.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: Transitioning from X/Twitter to new social media platforms
+Drivers: Théo Zimmermann (@Zimmi48)
+---
+
+## Summary
+
+This RFC proposes that as part of the actions to be taken in the context of the renaming of Coq to the Rocq Prover, a departure from X/Twitter should be announced and implemented. The rationale for this decision is based on the changing ownership and policy at X/Twitter, the current management of the X/Twitter account, the overall context of many academics and academic institutions leaving X/Twitter, and the opportunity presented by the renaming to move to more appropriate platforms.
+
+## Context and motivation
+
+The decision to leave X/Twitter is motivated by several factors:
+
+1. **Changing ownership and policy at X/Twitter:**
+  Since the acquisition of Twitter by Elon Musk, significant policy changes, including in moderation, promotion of misinformation, etc. have caused concern within the scientific and open-source communities. The Coq/Rocq team's values of openness, collaboration, and inclusivity, but also the academic and scientific communities' values, are increasingly at odds with the direction of the platform.
+
+1. **Costs associated with the renaming:**
+  Renaming Coq to the Rocq Prover triggers costs, including for the X/Twitter account, which will also need to be renamed. Doing the work of adapting the current CoqLang account to become a Rocq Prover X/Twitter account in the current context would send a political statement that the project does not care about the current direction the platform is taking. Disabling the existing account and stopping all use of X/Twitter is the only reasonable course of action in the context of the renaming.
+
+1. **Diminishing engagement and influence:**
+  [X/Twitter's relevance among academics is waning](https://www.insidehighered.com/news/tech-innovation/digital-publishing/2024/11/13/more-academics-flee-x-after-election), with many users and academic institutions (e.g., Aix-Marseille University, University of Strasbourg, University of Luxembourg, RWTH Aachen, Utrecht University, York University) having already left, and migrating to alternatives such as Mastodon and Bluesky. Furthermore, X/Twitter is already banned in Brazil, and an investigation against the platform by the EU commission is ongoing. In this context, staying active on X/Twitter is no longer a good use of community resources.
+
+1. **Current management of the X/Twitter account:**
+  The CoqLang account on X/Twitter was initially managed by three people. Out of these people, one has stopped being active in the Coq ecosystem, and I, Théo Zimmermann, have already stopped being active on X/Twitter at the time of Twitter's acquisition by Elon Musk. This means that there is only one person left to manage the account, with an irregular presence. Aiming to preserve the account would require adding more resources to this task, which is not desirable given the context.
```

Comment:
Can you explain more how my presence on X (previously Twitter) is "irregular"?

Reply:
I think it is meant as in "not regularly posting"? Which I don't think is meant as criticism of any kind.

Reply:
Indeed, very sorry Emilio that this may have sounded like a criticism. It was very far from it. I indeed meant irregular posting, and in my opinion, it is perfectly expected that the posting would be irregular if it is left on the shoulders of a single person.

Reply:
"irregular" as in "occurring at uneven or varying rates or intervals" rather than "contrary to the rules or to that which is normal or established".